src/fsharp/FSharp.Compiler.netcore.nuget/Microsoft.FSharp.Compiler.netcore.nuspec

```diff
@@ -16,33 +16,35 @@
-                <dependency id="System.Security.Cryptography.Csp" version="4.0.0-rc3-23805" />
-                <dependency id="System.Security.Cryptography.Primitives" version="4.0.0-rc3-23805" />
-                <dependency id="System.Threading.Tasks.Parallel" version="4.0.1-rc3-23805" />
-                <dependency id="System.Threading.Thread" version="4.0.0-rc3-23805" />
-                <dependency id="System.Threading.ThreadPool" version="4.0.10-rc3-23805" />
+                <dependency id="Microsoft.NETCore.Platforms" version="1.0.1-rc3-23910" />
+                <dependency id="NETStandard.Library" version="1.5.0-rc3-23910" />
+                <dependency id="System.Collections.Immutable" version="1.2.0-rc3-23910" />
+                <dependency id="System.Diagnostics.Process" version="4.1.0-rc3-23910" />
+                <dependency id="System.Diagnostics.TraceSource" version="4.0.0-rc3-23910" />
+                <dependency id="System.Linq.Expressions" version="4.0.11-rc3-23910" />
+                <dependency id="System.Linq.Queryable" version="4.0.1-rc3-23910" />
+                <dependency id="System.Net.Requests" version="4.0.11-rc3-23910" />
+                <dependency id="System.Reflection.Emit" version="4.0.1-rc3-23910" />
+                <dependency id="System.Reflection.Emit.ILGeneration" version="4.0.1-rc3-23910" />
+                <dependency id="System.Reflection.Metadata" version="1.3.0-rc3-23910" />
+                <dependency id="System.Runtime.InteropServices" version="4.1.0-rc3-23910" />
+                <dependency id="System.Runtime.InteropServices.PInvoke" version="4.0.0-rc3-23910" />
+                <dependency id="System.Runtime.Loader" version="4.0.0-rc3-23910" />
+                <dependency id="System.Security.Cryptography.Algorithms" version="4.1.0-rc3-23910" />
+                <dependency id="System.Security.Cryptography.Primitives" version="4.0.0-rc3-23910" />
+                <dependency id="System.Threading.Tasks.Parallel" version="4.0.1-rc3-23910" />
+                <dependency id="System.Threading.Thread" version="4.0.0-rc3-23910" />
+                <dependency id="System.Threading.ThreadPool" version="4.0.10-rc3-23910" />
+                <dependency id="Microsoft.DiaSymReader.PortablePdb" version="1.0.0-rc-60301" />
+                <dependency id="Microsoft.DiaSymReader" version="1.0.7" />
             </group>
         </dependencies>
     </metadata>
     <files>
-        <file src="fsc.exe"                 target="runtimes/any/native" />
-        <file src="FSharp.Compiler.dll"     target="runtimes/any/native" />
-        <file src="default.win32manifest"   target="runtimes/any/native" />
-        <file src="fsi.exe"                 target="runtimes/any/native" />
-        <file src="FSharp.Compiler.Interactive.Settings.dll" target="runtimes/any/native" />
+        <file src="fsc.exe"                 target="lib/DNXCore50" />
```

Comment:
Fsi and fsc can be left in "runtimes/any/native" This is because we don't want an app depending on this package to load the types in those exes when launching (or is there a reason we would?) Same for default.win32manifest as well.

From our conversation yesterday, it sounded like fsc looks for default.win32manifest and another file using BaseDirectory. Is that other file (FSharp.Core) in this package?